truffle/contracts/VaultNotary/VaultNotary.sol

```diff
@@ -0,0 +1,231 @@
+/// @author Jianwei Liu jliu@shipchain.io
+pragma solidity 0.5.11;
+
+import {Ownable} from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+
+/** @notice The VaultNotary contract is the contract for reading/writing the
+  * vault uri and hash, and controlling the permissions to those operations by vault
+  * owner.
+  */
+contract VaultNotary is Ownable {
+    struct Data {
```

Comment:
Read up on struct packing in solidity; the order of these variables matter.
https://solidity.readthedocs.io/en/v0.5.11/miscellaneous.html#layout-of-state-variables-in-storage

I think this order is probably fine - but I've been trying to leave comments showing which 32 byte 'slots' each variable go into.

The same applies to the contract variables below (isDeprecated/notaryMapping)